build/common.ps1

```diff
@@ -532,7 +533,7 @@ Function Invoke-ILMerge {
     Trace-Log 'Creating the ilmerged nuget.exe'
     $opts = , 'NuGet.exe'
     $opts += $buildArtifacts
-    $opts += "/out:$Artifacts\NuGet.exe"
+    $opts += "/out:$NuGetExeOutDev14\NuGet.exe"
```

Comment:
Great catch. Thanks--what I'll do is remove the changes to the build script for now. When we come to build both Dev14 and Dev15 targets (which isn't yet), I'll make sure the Dev14 nuget.exe + pdb stay in the artifacts root. The VSIX will need to be stored in separate output dirs though, but that's taken care of in VsExtension.csproj.